app/src/main/java/com/sdpteam/connectout/profileList/ProfilesContainerFragment.java

```diff
@@ -0,0 +1,52 @@
+package com.sdpteam.connectout.profileList;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ToggleButton;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.widget.Toolbar;
+import androidx.fragment.app.Fragment;
+
+import com.sdpteam.connectout.R;
+import com.sdpteam.connectout.drawer.DrawerActivity;
+import com.sdpteam.connectout.event.nearbyEvents.filter.EventsFilterDialog;
+import com.sdpteam.connectout.utils.DrawerFragment;
+import com.sdpteam.connectout.utils.WithFragmentActivity;
+
+public class ProfilesContainerFragment extends DrawerFragment {
+
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.activity_user_list, container, false);
+
+        Toolbar toolbar = view.findViewById(R.id.user_list_toolbar);
+        ToggleButton filterButton = view.findViewById(R.id.user_list_button);
+
+        ProfileListFragment listFragment = new ProfileListFragment();
+        ProfileFilterFragment filterFragment = new ProfileFilterFragment();
+
+        View.OnClickListener listener = v -> {
+            if (filterButton.isChecked()) {
```

Comment:
The filter in profiles view is not working because here it is a `ToggleButton` and in the drawer's view it is a `Button`.
Can you look into that ?

Reply:
Yes, the `Button` usage is now uniform. and the issue is fixed.